changelog/fragments/helm-metrics-addr.yaml

```diff
@@ -0,0 +1,14 @@
+entries:
+  - description: >
+      Added `--metrics-addr` flag to helm operator to make it configurable, and
+      changed the default from `:8383` to `:8080`
+
+    kind: "change"
+
+    # Is this a breaking change?
+    breaking: true
+
+    migration:
+      header: Default helm operator metrics port changed
+      body: >
+        To continue using port 8383, specify `--metrics-addr=:8383` when you start the operator.
```

Comment:
WDYT about we improve it and describe where the `--metrics-addr=:8383` need to be used (config/default, and config/manager/)?

Reply:
I disagree for a few reasons:
1. In general, it _should_ be obvious to most operator authors, and kubernetes users how to set flags on their operator.
1. In fairness, it is somewhat confusing when kustomize enters the picture, so I would propose that we write a general section in a doc somewhere about how to set flags on the manager when using the kustomize templates. This should probably be a kubebuidler doc.
1. There are lots of flags on operators. We shouldn't have to repeat ourselves in migration guides, docs, etc. anytime we discuss setting flags.

Reply:
Far enough.